nix/nixos/cardano-db-sync-service.nix

```diff
@@ -202,11 +202,16 @@ in {
           SNAPSHOT_SCRIPT=$( (yes phrase ||:) | cardano-db-tool prepare-snapshot --state-dir ${cfg.stateDir} | tail -n 1)
           res=$?
           set -e
+          SNAPSHOT="$(echo $SNAPSHOT_SCRIPT | cut -d " " -f3)"
```

Comment:
> `scripts/postgresql-setup.sh --create-snapshot db-sync-snapshot-schema-11-block-2917514-x86_64 /var/lib/cexplorer/37454316-6c80c64218-156.lstate`
so the cut will get the `db-sync-snapshot-schema-11-block-2917514-x86_64 ` part

https://input-output-rnd.slack.com/archives/GDB7LERDY/p1632170561466000?thread_ts=1632166000.461700&cid=GDB7LERDY